packages/yourloops/pages/login/login.tsx

```diff
@@ -192,6 +251,36 @@ function Login(props: RouteComponentProps): JSX.Element {
               </Button>
             </CardActions>
           </Card>
+          <Grid container>
+            <Grid item xs={12} className={classes.selection}>
+              <LanguageSelect />
+            </Grid>
+            <Grid item xs={4} className={classes.rightLink}>
+              <Link href={diabeloopUrls.getPrivacyPolicyUrL(i18n.language)}>
+                {t(diabeloopUrls.PrivacyPolicy)}
+              </Link>
+            </Grid>
+            <Grid item xs={4} className={classes.centeredLink}>
+              <Typography style={{ fontSize: "small" }}>
+                {`${t("Yourloops")} ${config.VERSION}`}
+              </Typography>
+            </Grid>
+            <Grid item xs={4} className={classes.leftLink}>
+              <Link href={diabeloopUrls.SupportUrl}>
+                {t(diabeloopUrls.Support)}
+              </Link>
+            </Grid>
+            <Grid item xs={6} className={classes.rightLink}>
+              <Link href={diabeloopUrls.getTermsUrL(i18n.language)}>
+                {t(diabeloopUrls.Terms)}
+              </Link>
+            </Grid>
+            <Grid item xs={6} className={classes.leftLink}>
+              <Link href={diabeloopUrls.getIntendedUseUrL(i18n.language)}>
+                {t(diabeloopUrls.IntendedUse)}
+              </Link>
+            </Grid>
+          </Grid>
```

Comment:
I would have put this part in another functional component (an JSX function), to keep the "HTML" part more readable.
You may put this function in the same file, like I've done in https://github.com/mdblp/blip/blob/215edb4bb8c66bedde0735d74f29682d2565c517/packages/yourloops/pages/hcp/team-card.tsx#L115 for example.

Reply:
i think the same, i was going to do it later, but you motivated me to do it now 😉